osgi/karaf-features/pom.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Hibernate Validator, declare and validate application constraints
+  ~
+  ~ License: Apache License, Version 2.0
+  ~ See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>hibernate-validator-osgi</artifactId>
+        <groupId>org.hibernate</groupId>
+        <version>5.2.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>hibernate-validator-osgi-karaf-features</artifactId>
+    <packaging>pom</packaging>
+
+    <name>Hibernate Validator Karaf features</name>
+    <description>Hibernate Validator features for Apache Karaf</description>
+
+    <properties>
+        <maven.deploy.skip>true</maven.deploy.skip>
```

Comment:
I guess we could. In this case it is simple enough. I am reluctant on Searhc, since the whole osgi-fying of the Lucene parts seems a bit hacky. We don't have this here, so I guess we could consider this also as a new feature to provide a Karaf feature.xml